### DISCUSSION TITLE
Disable window animations (minimize/unminimize)

### DISCUSSION THREAD

**Discussion by Anthropos888** [lawand-dot-io/taskbar#60] [Feature requests]
Is it possible to somehow disable the window animations when minimizing or unminimizing a window?

I have a M2 Pro but minimizing / unminimizing is so slow, the animation feels like 5 seconds. 
On a remote computer even longer.

I didn't find any system tweak to disable the animations.

Is it possible for you to disable the animations?
If so, a setting would be great where users can choose to enable/disable animations.

Thanks a lot for your great app.

**Comment by lawand-dot-io** [comment]
@Anthropos888 Hi there. You have three options:

1. Enable the "Clicking the focused task hides its app" option so that when you click a window in Taskbar it's hidden instead of minimized. This has no animation but the downside is that the whole app is hidden and not just one window.

2. In system preferences, select the "scale effect" for the minimize animation, I personally use that and find it faster than the default genie effect.

3. One thing I noticed is that when I enable the "Fully hide dock" option in Taskbar settings, sometimes there's a macOS bug where the minimize animation no longer happens. This isn't always consistent and also has the downside of disabling the dock.

Hope this helps and hope you continue to find Taskbar useful.

Cheers :)

**Reply by ChrissyTopal** [reply]
Thank you for redirecting me here. I did not know about the scale effect. I think this will do.
